### ISSUE TITLE
Theme using InteractiveAuto

### ISSUE THREAD

**Issue by manuelelucchi** [ant-design-blazor/ant-design-blazor#3792]
Hi,
I'm experimenting with Ant and InteractiveAuto in the .NET 8 Blazor WebApp. Everything seems to work fine and fast, but I noticed that if I set the Dark Theme with the ConfigProvider (let's say it's fixed as dark), it first loads the white theme (supposedly the SSR?) and then switches to dark. This is not great to see. Is there a way to anticipate the theme loading while it's doing the SSR part?

**Comment by ElderJames** [comment]
Hi @manuelelucchi , this is a common problem of SSR, that is, the corresponding theme needs to be output in the SSR stage. AntBlazor is not currently supported. You can try using JS to fix the theme. Just load different files.

- dark:
```
_content/AntDesign/css/antd-design-blazor.dark.css
```
- default:
```
_content/AntDesign/css/antd-design-blazor.css
```

**Comment by manuelelucchi** [comment]
Makes sense, thank you. Can i use the same concept for the primary color? This is more of broader question, but since we don't have the css variables yet

**Comment by ElderJames** [comment]
We have css variables, _content/AntDesign/css/antd-design-blazor.variables.css

**Comment by manuelelucchi** [comment]
Oh, is there an example somewhere? It's really an obscure topic
